### DISCUSSION TITLE
Upgrade error

### DISCUSSION THREAD

**Discussion by tellustheguru** [kimai/kimai#4315] [On-Premise: Installation & Updates] [answered]
### Describe the issue

$ sudo -u www-data composer inssudo -u www-data composer install --optimize-autoloader -n

Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Nothing to install, update or remove
Generating optimized autoload files
127 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 1
!!  PHP Notice:  Found invalid "kimai" configuration: Unrecognized option "days" under "kimai.calendar.businessHours". Available options are "begin", "end". in /www-root/kimai2/src/DependencyInjection/AppExtension.php on line 29
!!  
!!  In ArrayNode.php line 304:
!!                                                                                 
!!    Unrecognized option "days" under "kimai.calendar.businessHours". Available   
!!    options are "begin", "end".                                                  
!!                                                                                 
!!  
!!  
Script @auto-scripts was called via post-install-cmd

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

1,30.11

### How do you run Kimai?

Other (please mention below)

### Which PHP version are you using?

8.1

**Comment by kevinpapst** [comment, marked as answer]
See https://github.com/kimai/kimai/blob/main/UPGRADING.md

Remove your local.yaml and retry.

**Reply by tellustheguru** [reply]
Thanks, everything works!!!
